### DISCUSSION TITLE
Search should highlight the keyword in the relevant notes

### DISCUSSION THREAD

**Discussion by equasman** [standardnotes/forum#2717] [Suggestions]
Hi,

Consider a real case scenario: I have many large text notes. Some of them even contain the same word multiple time.

Now suppose I searched for the word "standard" in my Standard Notes. More than one notes contain the word "standard". Few of them even have it multiple times.

My life would be easier if Standard Notes showed me where this word is; it can highlight this word in the relevant notes. Even better would be to show the search results like 1 of 30 or the likes. If I see the first instance of the word I searched for, I want to know how many more are there.

Please consider implementing this feature.

Thank you for your great work.

**Comment by JaspalSuri** [comment]
Hi @equasman,

Currently the desktop app highlights the words in the notes that appear in the search results after you open them individually, but I'll be sure to include the part about adding the amount of times the requested query appears in each note in my report to the dev team. 

Thanks for your feedback!

**Comment by equasman** [comment]
But it isn't available for either the web or mobile version.
Please make it available for the basic users.

**Comment by JaspalSuri** [comment]
It's not a feature that's restricted to users who subscribe to [Extended](https://standardnotes.org/extensions/), but it doesn't work on the mobile app yet. I'll add that to my report. I'll check in with our desktop & web dev to see why it doesn't work on the web app.

**Comment by equasman** [comment]
Thanks a lot.

**Comment by arielsvg** [comment]
Hey @equasman, on desktop we are leveraging native capabilities that we don't have access to on other platforms. We would like to introduce highlighting on all platforms but that's a bigger project in the backlog :)

**Comment by equasman** [comment]
By "leveraging native capabilities", I think you meant pressing **Ctrl+F** and use the web browser's search feature. It does the job. But having the capability in Standard Notes would be great.


> We would like to introduce highlighting on all platforms but that's a bigger project in the backlog :)

Glad to know we'll have this and many more in the future. Thanks for your work on Standard Notes.
